### PR TITLE
Backports for v2.1.1

### PR DESCRIPTION
**Pull request by mheon**
And release notes for said release

### CODE REVIEW

**Comment by mheon** [comment]
@vrothberg @rhatdan @baude @TomSweeneyRedHat PTAL

**Comment by rhatdan** [comment]
LGTM

File: pkg/domain/infra/tunnel/events.go [hunk 1]
```diff
@@ -2,8 +2,10 @@ package tunnel
 
 import (
 	"context"
+	//	"fmt"
```
Comment (jwhonce) [inline pkg/domain/infra/tunnel/events.go:5]:
```suggestion
```
Should just delete, yes?

**Comment by mheon** [comment]
Anybody want to merge this?

**Review by jwhonce — APPROVED** [review]
LGTM with one very small nit

**Comment by jwhonce** [comment]
/lgtm
nit will be fixed on master by @mheon

**Pull request merged**